src/libraries/Common/tests/System/Security/Cryptography/MLKemNotSupportedTests.cs

```diff
@@ -64,5 +64,25 @@ public static void ImportDecapsulationKey_NotSupported(MLKemAlgorithm algorithm)
                 algorithm,
                 new Span<byte>(new byte[algorithm.DecapsulationKeySizeInBytes])));
         }
+
+        [Fact]
+        public static void ImportPkcs8PrivateKey_NotSupported()
+        {
+            Assert.Throws<PlatformNotSupportedException>(() => MLKem.ImportPkcs8PrivateKey(
+                MLKemTestData.IetfMlKem512PrivateKeySeed));
+
+            Assert.Throws<PlatformNotSupportedException>(() => MLKem.ImportPkcs8PrivateKey(
+                new ReadOnlySpan<byte>(MLKemTestData.IetfMlKem512PrivateKeySeed)));
+        }
+
+        [Fact]
+        public static void ImportEncryptedPkcs8PrivateKey_NotSupported()
+        {
+            Assert.Throws<PlatformNotSupportedException>(() => MLKem.ImportEncryptedPkcs8PrivateKey(
+                MLKemTestData.EncryptedPrivateKeyPassword, MLKemTestData.IetfMlKem512PrivateKeySeed));
+
+            Assert.Throws<PlatformNotSupportedException>(() => MLKem.ImportEncryptedPkcs8PrivateKey(
+                MLKemTestData.EncryptedPrivateKeyPasswordBytes, MLKemTestData.IetfMlKem512PrivateKeySeed));
```

Comment:
The PNSE will fire whether the data is valid or not, but I assume you meant `IetfMlKem512EncryptedPrivateKeySeed` instead of the unencrypted one?

Reply:
Yeah... that's going to bug me... so let's fix it.